tasks/patternpack.js

```diff
@@ -134,6 +139,24 @@ module.exports = function (grunt) {
     }
   }
 
+  function ensureFilesExist(options) {
+    var path = require("path");
+
+    // Create core css file if it does not exist
+    var coreCssPath = options.assets + "/" + options.css.preprocessor + "/" + options.css.fileName + "." + options.css.preprocessor;
+    if (!fs.existsSync(coreCssPath)) {
+      fs.writeFileSync(coreCssPath, '@import "_patternpack-patterns"');
+    }
+
+    // Create the dirctories for the pattern structure if they do not exist
+    _.each(options.patternStructure, function(pattern) {
+      var patternPath = options.src + "/" + pattern.path;
+      if (!fs.existsSync(patternPath)) {
+        fs.mkdir(patternPath);
+      }
+    });
```

Comment:
Once we add things like this we will need to change our strategy a bit.  If we want full templates, then we should create a `templates` folder that has the placeholder templates, and we should copy the file templates into the project, when the files are missing.